hooks/pre_commit/05-puppet_certs_exist.rb

```diff
@@ -0,0 +1,27 @@
+def error(message)
+  say message
+  logger.error message
+  kafo.class.exit 101
```

Comment:
I'm not certain where the bug is, but I get this error:

<pre>
/usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/logger.rb:104:in `error': wrong number of arguments (3 for 1) (ArgumentError)
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/logger.rb:104:in `send'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/logger.rb:104:in `dump_buffer'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/logger.rb:104:in `each'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/logger.rb:104:in `dump_buffer'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/logger.rb:103:in `each'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/logger.rb:103:in `dump_buffer'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/logger.rb:94:in `dump_errors'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/logger.rb:99:in `dump_errors'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/exit_handler.rb:26:in `exit'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/kafo_configure.rb:125:in `exit'
        from /usr/share/foreman-installer/hooks/pre_commit/05-puppet_certs_exist.rb:4:in `error'
        from /usr/share/foreman-installer/hooks/pre_commit/05-puppet_certs_exist.rb:22:in `load'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/hook_context.rb:13:in `instance_exec'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/hook_context.rb:13:in `execute'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/hooking.rb:50:in `execute'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/hooking.rb:48:in `each'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/hooking.rb:48:in `execute'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/kafo_configure.rb:105:in `execute'
        from /usr/lib/ruby/gems/1.8/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
        from /usr/lib/ruby/gems/1.8/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `run'
        from /usr/lib/ruby/gems/1.8/gems/kafo-0.6.11/lib/kafo/kafo_configure.rb:119:in `run'
        from /usr/sbin/foreman-installer:18
</pre>